src/coverage.ts

```diff
@@ -152,6 +266,12 @@ function deactivateCoverage() {
   // delete all decorations
   if (uncoveredDecorationType) uncoveredDecorationType.dispose();
```

Comment:
try iterating over a list and calling the condition followed by dispose() to avoid repetition